packages/hdwallet-keepkey/src/eos.ts

```diff
@@ -0,0 +1,204 @@
+import * as Core from '@shapeshiftoss/hdwallet-core'
+
+import { KeepKeyTransport } from './transport'
+
+import {
+  EosGetPublicKey,
+  EosPublicKey,
+  EosPublicKeyKindMap,
+  EosSignTx,
+  EosTxHeader,
+  EosTxActionRequest,
+  EosTxActionAck,
+  EosActionCommon,
+  EosPermissionLevel,
+  EosAsset,
+  EosActionTransfer,
+  EosSignedTx,
+} from '@keepkey/device-protocol/lib/messages-eos_pb'
+import {
+  MessageType
+} from '@keepkey/device-protocol/lib/messages_pb'
+
+import { cloneDeep } from 'lodash'
+
+var Long = require("long")
+
+export function eosGetAccountPaths (msg: Core.EosGetAccountPaths): Array<Core.EosAccountPath> {
+  return [{
+    addressNList: [ 0x80000000 + 44, 0x80000000 + Core.slip44ByCoin('Eos'), 0x80000000 + msg.accountIdx, 0, 0 ]
+  }]
+}
+
+
+function charToSymbol(c:string) : number {
+  if (c.charCodeAt(0) >= "a".charCodeAt(0) && c.charCodeAt(0) <= "z".charCodeAt(0)) {
+    return ((c.charCodeAt(0) - "a".charCodeAt(0)) + 6)
+  }
+  if (c.charCodeAt(0) >= "1".charCodeAt(0) && c.charCodeAt(0) <= "5".charCodeAt(0)) {
+    return ((c.charCodeAt(0) - "1".charCodeAt(0)) + 1)
+  }
+  return 0
+}
+
+function nameToNumber(name:string) : string {
```

Comment:
Whats this do?

Reply:
Converts the alphanumeric name of an action to a format that is appropriate for signing. This was derived from the python-keepkey tests for eos that came from trezor.